src/components/views/UserPage.js

```diff
@@ -1,6 +1,67 @@
-const UserPage = () => {
-    return <div>UserPage</div>
+import React, {useEffect, useState} from "react";
+import Header from "./Header";
+import {api} from 'helpers/api';
+import BaseContainer from "../ui/BaseContainer";
+import 'styles/views/UserPage.scss'
+
+
+const PlayerProfile = ({user}) =>(
+    <div className="userPage container">
+        <ul className="userPage player-info-container">Username: {user.username}</ul>
+        <ul className="userPage player-info-container">Name: {user.name}</ul>
+        <ul className="userPage player-info-container">Gender: {user.gender}</ul>
+        <ul className="userPage player-info-container">Birthday: {displayDate(user.birthday)}</ul>
+    </div>
+)
+const displayDate = (date) => {
+    if(date){
+        return date.substring(8,10)+"-"+date.substring(5,7)+"-"+date.substring(0,4);
+    }
+    else{
+        return "Date not set yet";
+    }
 }
 
+const UserPage = () =>{
+    const[user, setUser] = useState(null);
+
+    useEffect(() => {
+
+        async function getUser(){
+            const id = localStorage.getItem("id")
+            const response = await api.get('/users/'+id)
+            setUser(response.data);
+            console.log(response)
+        }
+        getUser()
+
+    }, []);
+    let profile = (
```

Comment:
What is the point in having a placeholder? I'd rather no data instead of wrong data in case of failure.

Reply:
Done